packages/chat/src/components/Button/styles.css.ts

```diff
@@ -1,10 +1,15 @@
+import { style } from '@vanilla-extract/css';
 import { recipe } from '@vanilla-extract/recipes';
 
 import { PALETTE } from '@/styles/colors.css';
 import { SIZES } from '@/styles/sizes';
 
 import { buttonReset } from './reset.css';
 
+export const roundButton = style({
+  borderRadius: '999px !important',
```

Comment:
why is the `!important` needed?

Reply:
It's not needed... :)

While i was working, did that just as a test, cause i wasn't seeing it, but then moved it anyway into the `buttonStyles` variant. You can see it there (without `!important`)